stl/inc/string

```diff
@@ -481,7 +481,7 @@ _Elem* _UIntegral_to_buff(_Elem* _RNext, _UTy _UVal) { // format _UVal into buff
     return _RNext;
 }
 
-template <class _Elem, class _Ty>
+template <class _Elem, class _Ty, enable_if_t<is_signed_v<_Ty>, int> = 0>

```

Comment:
The callers below know whether `_Ty` is signed; why not give these two functions different names and avoid the cost of overload resolution?

Reply:
Technically, I don't know if `char` is signed. But it's usually signed (without the `/J` abomination-option), and the signed overload still handles unsigned types.

Reply:
Never mind, `char` is never the value type.